### ISSUE TITLE
Usages of a parent group are not working if a sub-group is present

### ISSUE THREAD

**Issue by ElviaSober** [F43nd1r/gradle-version-catalogs-intellij-plugin#7]
I separate the identifiers by dash in the toml file. When I have there one alias named `spring-boot` and another one named `spring-boot-starter-actuator`, then `spring-boot` shows no usages and the navigation from a usage into it also doesn't work.

Example of the toml file:
```
[versions]
...

[libraries]
spring-boot = { module = "org.springframework.boot:spring-boot" }
spring-boot-starter-actuator = { module = "org.springframework.boot:spring-boot-starter-actuator" }
```
Using version 1.1.0 of the plugin :)

**Comment by F43nd1r** [comment]
Fix released as 1.2.0 (pending jetbrains approval)

**Comment by ElviaSober** [comment]
Thanks for the fast fix! :)
